examples/gitlab/deploy-to-a-single-device.gitlab-ci.yml

```diff
@@ -0,0 +1,71 @@
+#
+# The example shows how to deploy a Mender artifact to a single device.
+#
+# 'MENDER_SERVER_ACCESS_TOKEN' secret is required to be set in GitLab repo settings
+#
+
+stages:
+  - build
+  - publish
+  - deploy
+
+include:
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-artifact-build.gitlab-ci.yml'
+    ref: 'master'
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-artifact-upload.gitlab-ci.yml'
+    ref: 'master'
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-deployment-create.gitlab-ci.yml'
+    ref: 'master'
+
+variables:
+  MENDER_ARTIFACT_NAME: artifact.mender # Mender Artifact file name in the `MENDER_ARTIFACTS_FOLDER` folder
+  MENDER_RELEASE_NAME:  auth_keys-1.0   # The name of Mender release
+
+#
+# Builds Mender demo single-file artifact and stores '.mender' binary
+# to MENDER_ARTIFACTS_FOLDER folder
+#
+build:mender-artifact:
+  stage: build
+  extends: .mender:build:artifact
+  script:
+    - echo /home/root/.ssh > dest_dir
+    - echo authorized_keys > filename
+    - echo key > authorized_keys
+    - |
+      mender-artifact write module-image \
+        -T single-file \
+        --device-type raspberrypi4 \
+        -o ${MENDER_ARTIFACTS_FOLDER}/${MENDER_ARTIFACT_NAME} \
+        -n ${MENDER_RELEASE_NAME} \
+        --software-name authorized_keys \
+        --software-version 1.0 \
+        -f dest_dir \
+        -f filename \
+        -f authorized_keys
+
+#
+# Uploads built artifact from MENDER_ARTIFACTS_FOLDER folder to Mender server
+# on merge event to 'main' branch
+#
```

Comment:
The per-job documentation is inconsistent. Some examples have more verbose comments than others, some jobs are missing doc, some like this one is not completely correct(*). I suggest we simplify or ever remove all of them. By the name it should be obvious what they are doing

(*) Incorrect in the sense that only one Artifact is supported:
```suggestion
#
# Uploads built artifact ${MENDER_ARTIFACTS_FOLDER}/${MENDER_ARTIFACT_NAME} to Mender server
# on merge event to 'main' branch
#
```

Reply:
Okey.